src/agent/agent_cli.js

```diff
@@ -204,7 +204,7 @@ class AgentCLI {
     }
 
     rename_agent_storage(mount_points) {
-        if (os.type() === 'Darwin') return; // skip rename in mac os
+        if (process.platform === 'darwin') return; // skip rename in mac os
```

Comment:
instead of repeating this literal strings of platform names in the code,
can we expose these booleans from os_utils:
https://github.com/noobaa/noobaa-core/blob/22e1a6f0c36ddc8fdfc11ddf9ba5cae6e29a5df1/src/util/os_utils.js#L32-L37

Also good chance to merge with these in fs_utils:
https://github.com/noobaa/noobaa-core/blob/22e1a6f0c36ddc8fdfc11ddf9ba5cae6e29a5df1/src/util/fs_utils.js#L17-L18